sites/svelte.dev/src/routes/_components/Supporters/index.svelte

```diff
@@ -86,11 +93,11 @@
 	}
 
 	.contributors .supporter {
-		background-image: url(./contributors.jpg);
+		background-image: -webkit-image-set(url(./contributors.jpg));
```

Comment:
Are these needed along with the inline style?

Reply:
How does that work? IIUC the inline style will just override the background-image property so it'll be like the value from the CSS isn't there whatsover. It could maybe fallback if it encountered some unsupported css value or css function, but that won't be the case since it's an unsupported image format, not an unsupported CSS value.

Reply:
Oh ur right! I confused this with style tag's CSS progressive enhancements So this in fact wouldn't work.

Which means we can't have fallback here.

Hmm I'd say its fine, webp has 96% support https://caniuse.com/webp